### DISCUSSION TITLE
Water Heater: Add manual mode

### DISCUSSION THREAD

**Discussion by sebmalissard** [home-assistant/architecture#873] [Entity Models]
Hi, 

I have a water heater with 2 main modes:

- Eco (auto, the system automatically adjusts the quantity of water to heat)
- Manual (the user manualy selects the quantity of water to heat)

The Eco mode is correctly displayed in the operation_mode of the widget water_heater, but when I select manual mode (on the water heater IHM) the operation_mode widget always raised "disabled" even if the water header is actif.

So, my proposition is to add a 'manual' state in https://developers.home-assistant.io/docs/core/entity/water-heater#states for this kind of water header. For that, I have started to draft a MR: https://github.com/home-assistant/core/pull/87508

And maybe a second step in another MR, would be to define new method to allow use to define the quantity of water (with my water heater overkiz API this match with the quantity of shower request/available) to heat when we are in manual mode.

**Comment by make-all** [comment]
Can you not just use "electric" or "gas" (as appropriate) for the manual mode?  The objective should be to get your device to fit into the HA entity model, not to expand the HA entity model to exactly match all different terminology used by different manufacturers.
